package-lock.json

```diff
@@ -1,8 +1,10957 @@
 {
   "name": "eslint-github-bot",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This change produces the following warning in Lint and Test jobs, which are run on Node 12 (presumably with npm v6):

```
Run npm install
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
```

[documentation](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion) for `lockfileVersion` says that `lockfileVersion@2` is backwards compatible to v1. I believe this means that this `package-lock.json` file will be correctly read and used by npm v6 (we are using npm v6 in the production environment?), but from now on we should probably only use npm 7+ to update `package-lock.json`.

Reply:
This was already running on npm 7+

Reply:
Yes, but this is most likely the first time the lock file was updated with npm 7+. Since it's now in the new format, we probably should never use npm < 7 to update that file again.